python/instrumentation/openinference-instrumentation-llama-index/tests/openinference/instrumentation/llama_index/test_callback.py

```diff
@@ -227,15 +218,6 @@ def _check_spans(
         assert synthesize_span.status.status_code == trace_api.StatusCode.OK
         assert not synthesize_span.status.description
         assert synthesize_attributes.pop(OUTPUT_VALUE, None) == answer
-    elif (
-        # FIXME: currently the error is propagated when streaming because we don't rely on
-        # `on_event_end` to set the status code.
-        status_code == 400 and is_stream
-    ):
-        assert synthesize_span.status.status_code == trace_api.StatusCode.ERROR
```

Comment:
maybe comment out and leave a note. instead of deleting

Reply:
addressed in 00dd9eee4f2fd9738953e67720f89cac00efbf7b